src/app/common/hooks/balance/btc/use-btc-crypto-currency-asset-balance.ts

```diff
@@ -0,0 +1,11 @@
+import { useNativeSegwitBalance } from '@app/query/bitcoin/balance/btc-native-segwit-balance.hooks';
+import { useCurrentAccountNativeSegwitSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+export function useBtcCryptoCurrencyAssetBalance() {
+  const currentBtcSigner = useCurrentAccountNativeSegwitSigner();
+  // TODO: it would be better if we could skip providing the empty string to this hook.
```

Comment:
Can we refactor this now?

Reply:
I tried it back then and it seemed like it would require quite a handful of code changes. I will see if i could do it now with a minimal amount of changes, if not will leave it for later

Reply:
OK great, thanks!